### DISCUSSION TITLE
ImageMagick crashing with no message

### DISCUSSION THREAD

**Discussion by felipesolanet** [ImageMagick/ImageMagick#7656] [Help]
### ImageMagick version

7.1.1

### Operating system, version and so on

windows 64 bit

### Description

I am trying to convert the pdf into a png image and it just crashes with no message.  Attached is the PDF I am trying to convert, and below is the command.
magick convert "C:\Temp\Page1.pdf[0]" -debug "All" -background white -alpha remove -alpha off "C:\Temp\myImage.png"
[Page1.pdf](https://github.com/user-attachments/files/17193433/Page1.pdf)

**Comment by fmw42** [comment]
On IM 7, use magick, not magick convert. See if that works.

What do you get from 

`magick -version`

Should you not be using magick.exe on Windows?

**Reply by fmw42** [reply]
It works fine for me on IM 7.1.1.38 on Mac OSX Ventura.

`magick page1.pdf -background white -alpha remove -alpha off x.png`

![x](https://github.com/user-attachments/assets/a34b7c01-6d00-441a-86e6-b7dd004e237d)

**Comment by felipesolanet** [comment]
[page1.pdf](https://github.com/user-attachments/files/17194473/page1.pdf)

Thank you for responding fmw42.  How about with this last attached pdf?  This one is crashing IM when I try.
My version command is attached as an image as well.
![image](https://github.com/user-attachments/assets/84552352-2da1-49f0-a06d-0f46791de187)

I tried without the convert but it is still crashing IM

**Reply by dlemstra** [reply]
Have you checked if you can reproduce this with the most recent version of ImageMagick?

**Reply by fmw42** [reply]
You are using a rather old version of IM.  Perhaps it has a bug.  Can you try the latest IM?  I tried with page1.pdf and it works fine for me.

**Comment by snibgo** [comment]
IM delegates the rasterization of PDF files to Ghostscript. Do you have Ghostscript installed?

What is the text output from:
```
magick -verbose page1.pdf info:
```

**Comment by felipesolanet** [comment]
Thank you for your responses fmw42 and snibgo
I made some progress.  The original PDF has two pages, which I split in 2 pdf files page1 and page2.  The page that crashes in page1, and it does so together or separate.  Page2 converts fine also in both scenarios together or separate. So there is something strange with page1, perhaps something corrupted.
I will try now upgrading imageMagick

**Comment by snibgo** [comment]
> I will try now upgrading imageMagick

Also consider upgrading Ghostscript.
